typings/pinejs-client-core.d.ts

```diff
@@ -485,9 +505,19 @@ export interface SubscribeParamsWithId<T> extends ParamsObjWithId<T> {
 	pollInterval?: number;
 }
 
+export type ConstructorParams = Pick<
+	ParamsObj<unknown>,
+	(typeof validParams)[number]
+>;
+
 export interface Pine<ResourceTypeMap extends object = object> {
 	apiPrefix: string;
-	delete<T>(params: ParamsObjWithId<T> | ParamsObjWithFilter<T>): Promise<'OK'>;
+	passthrough: AnyObject;
```

Comment:
What does passthrough means in the context of the SDK?

Reply:
We are not using it in the SDK atm, but if we don't blindly copy it from pinejs-client-core, this will be an extra missing property of the base class typings that TS will be complaining about.

Reply:
Effectively, this while PR is copying typings from pinejs-client-core, until we completely drop the custom typings.
So all these props are going to be added anyway once the custom typings get dropped.